history/taskana-simplehistory-provider/pom.xml

```diff
@@ -64,21 +64,9 @@
 			<scope>test</scope>
 		</dependency>
 		<dependency>
-			<groupId>org.junit.vintage</groupId>
-			<artifactId>junit-vintage-engine</artifactId>
-			<version>${version.junit.jupiter}</version>
-			<scope>test</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.powermock</groupId>
-			<artifactId>powermock-api-mockito2</artifactId>
-			<version>${version.powermock}</version>
```

Comment:
is this property in use? I don't think so. - Please verify and remove it in the taskana-parent.pom. Thanks!